Intersect.Client/Localization/Strings.cs

```diff
@@ -76,6 +76,62 @@ public static string FormatQuantityAbbreviated(long value)
             }
         }
 
+        public static string FormatTimeAbbreviated(long milliseconds)
+        {
+            if (milliseconds == 0)
+            {
+                return string.Empty;
+            }
+
+            double timeValue;
+            string postfix;
+
+            switch (milliseconds)
+            {
+                // seconds
+                case <= 60000:
+                    timeValue = milliseconds / 1000.0;
+                    postfix = Time.Second;
+                    break;
+
+                // minutes
+                case <= 600000:
+                    timeValue = milliseconds / 60000.0;
+                    postfix = Time.Minute;
+                    break;
+
+                // hours
+                case <= 36000000:
+                    timeValue = milliseconds / 3600000.0;
+                    postfix = Time.Hour;
+                    break;
+
+                // days
+                case <= 864000000:
+                    timeValue = milliseconds / 86400000.0;
+                    postfix = Time.Day;
+                    break;
+
+                // weeks
+                case <= 6048000000:
+                    timeValue = milliseconds / 604800000.0;
+                    postfix = Time.Week;
+                    break;
+
+                default:
+                    return "OOB";
+            }
```

Comment:
Switch on `<timeSpan>.Ticks` and cases should use `TimeSpan.TicksPer<unit>` for the values

Units should be `ns`, `μs`, `ms`, `s`, `m`, `d`, `w`, `y`

There is no constant for nanoseconds, weeks, or years so nanoseconds would be `<= TimeSpan.TicksPerMicrosecond`, days is `<= TimeSpan.TicksPerDay * 7` and weeks is ` <= TimeSpan.TicksPerDay * 365` -- years is `default` case